apps/studio/src/routes/editor/Toolbar/index.tsx

```diff
@@ -45,6 +45,20 @@ const TOOLBAR_ITEMS: {
         disabled: false,
         draggable: true,
     },
+    {
```

Comment:
@drfarrell were we hoping to have a dropdown list instead of adding 2 new buttons? The recall the design being slightly different

Reply:
that's right – @caleb8315 you originally mocked this up in one of your first versions, but we were imagining the UI looking like this with the up caret next to the div. I made some slight UI adjustments to the original mockup

<img width="544" alt="image" src="https://github.com/user-attachments/assets/8c9df077-ae74-4afa-88f1-c91085d9354e">

It was very much inspired by what Figma has.

<img width="274" alt="Screenshot 2024-11-15 at 1 04 03 PM" src="https://github.com/user-attachments/assets/eede3b92-14ba-4dd6-ba96-b540ded58482">